src/openklant/accounts/management/commands/createinitialsuperuser.py

```diff
@@ -33,7 +33,7 @@ def handle(self, *args, **options):
         User.objects.create_superuser(username=username, email=email, password=password)
 
         try:
-            link = f'{settings.ALLOWED_HOSTS[0]}{reverse("admin:index")}'
+            link = f"{settings.ALLOWED_HOSTS[0]}{reverse('admin:index')}"
```

Comment:
Could you add a file `.git-blame-ignore-revs` in the root of the repo and add the full commit hash of the commit that applies the formatting changes? That way git blame will exclude it